udfs/community/README.md

```diff
@@ -733,6 +734,37 @@ SELECT bqutil.fn.cw_disjoint_partition_by_regexp(5, 'B@5#', '(?:A@\\d+#)+(?:B@\\
 []
 ```
 
+### [cw_parse_timestamp(timeString STRING, formatString STRING)](cw_parse_timestamp.sqlx)
+Parses a timestamp string according to a specified format string. Returns a TIMESTAMP value.
```

Comment:
```suggestion
Parses a timestamp string according to a specified format string. Returns a DATETIME value.
```

Reply:
Modify description in both README and the UDF description to mention that DATETIME value will always be in UTC time zone